### ISSUE TITLE
Navigator inputs are mislabeled?

### ISSUE THREAD

**Issue by Cubistguitar** [patheros/JPManuals#10]
It appears to me that the Orange Input A should correspond to the Orange numbers and the Purple Input B should correspond to the Purple numbers, but presently my version is reversed. Orange numbers follow the Purple input. Windows 10 VCV RACK 2 Pro 2.4.1

**Comment by patheros** [comment]
Good catch. Yes they are backwards. I will get this fixed in the next build.

**Comment by patheros** [comment]
Fixed now with version 2.1
